File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # Store the value, and the other value needed to be valid
        val_map = {}
        for i in range(len(nums)):
            # Add val and target

            # Try and see if this value is a target
            if nums[i] in val_map.keys():
                return [val_map[nums[i]], i]
            val_map[target - nums[i]] = i


            

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        // Double the array?

        int[] toReturn = new int[nums.length * 2];

        for (int j = 0; j < nums.length; j++)
        {
            toReturn[j] = nums[j];
            toReturn[j+nums.length] = nums[j];
        }
    

        return toReturn;

    }
}